lib/web/ui/usercontext.go

```diff
@@ -124,9 +126,11 @@ func NewUserContext(user services.User, userRoles services.RoleSet) (*UserContex
 	userAccess := newAccess(userRoles, ctx, services.KindUser)
 	tokenAccess := newAccess(userRoles, ctx, services.KindToken)
 	nodeAccess := newAccess(userRoles, ctx, services.KindNode)
+	appServerAccess := newAccess(userRoles, ctx, "TODO: ADD THE RIGHT RESOURCE")
```

Comment:
> TODO: ADD THE RIGHT RESOURCE

Missing TODO

Reply:
@benarent do we want to be able to restrict access to applications tab? What about Nodes tab?

Reply:
Yes, let's do it for customers who don't want to show Apps at all. We should also do the same for nodes tab.